audit/projects/k8s-staging-gateway-api/buckets/k8s-staging-gateway-api-gcb/iam.json

```diff
@@ -3,27 +3,27 @@
     {
       "members": [
         "group:k8s-infra-artifact-admins@kubernetes.io",
-        "projectEditor:k8s-staging-service-apis",
-        "projectOwner:k8s-staging-service-apis"
+        "projectEditor:k8s-staging-gateway-api",
+        "projectOwner:k8s-staging-gateway-api"
```

Comment:
lots of files changing but these are expected:
- moves of `service-apis` -> `gateway-api`
- additions of `gateway-api` resources
- deletions of `service-apis` resources

this was https://github.com/kubernetes/k8s.io/pull/1954 getting deployed